yt/sample_data/api.py

```diff
@@ -112,9 +113,8 @@ def get_data_registry_table():
     )
 
     # load local data
-    pooch_json = json.load(
-        pkg_resources.resource_stream("yt", "sample_data_registry.json")
-    )
+    with pkg_resources.resource_stream("yt", "sample_data_registry.json") as fh:
```

Comment:
Since we are refactoring, is it worth getting rid of the loading completely and using pd to parse it from JSON to begin with?  I believe we could just use `pd.read_json` on the object returned by the resource stream.

(Not necessary)

Reply:
I haven't found a way to do this. The format used for this json file (mandated by pooch) is a little tricky to get into pandas, hence my 3 lines where one _should_ be enough. I'm happy to integrate a change here if anyone can figure it out though

Reply:
nah